### PR TITLE
Consolidate code checking in single script

### PR DESCRIPTION
**Pull request by nakulkorde**

### CODE REVIEW

File: scripts/test-codingstyle-all.sh [hunk 1]
```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Function defintion to check cpp coding style
+check_c () {
+    pushd $(dirname $0)
+
+    CHECKPATCH=checkpatch.pl
+    FILES=$(find ../libopae ../common/include/opae ../ase/api/src ../ase/sw ../tools/base/fpgaconf ../tools/base/fpgad \( -iname "*.c" -or -iname "*.h" \) -and \( ! -name "srv.c" \) -and \( ! -path "../libopae/src/usrclk/*" \) -and \( ! -path "../common/include/opae/cxx/*" \) )
```
Comment (tswhison) [inline scripts/test-codingstyle-all.sh]: Just a heads-up that libopae/src is going away once we merge feature/plugin-api back to master.  We'll have to scrub things then.

Comment (nakulkorde) [reply scripts/test-codingstyle-all.sh]: yes, that's right. I will update it once the plugin api is merged into master.

Comment (r-rojo) [reply scripts/test-codingstyle-all.sh]: checkpatch should only run on C files.

Comment (nakulkorde) [reply scripts/test-codingstyle-all.sh]: checkpatch is only looking at C files and header files.

Comment (r-rojo) [reply scripts/test-codingstyle-all.sh]: sorry, I missed the `!`

**Pull request merged**